contributors.txt

```diff
@@ -139,3 +139,4 @@ YYYY/MM/DD, github id, Full name, email
 2020/09/09, adamshakhabov, Adam Shakhabov, theadamo86(at)gmail.com
 2020/09/13, hoborm, Mihaly Hobor, mail(at)hobor.xyz
 2020/07/11, blitzmann, Ryan Holmes, holmes.ryan.90(at)gmail.com
+2020/10/03, m417z, ,
```

Comment:
Name and email is required (nor use special or ask someone else to use the patch)